saleor/product/models/base.py

```diff
@@ -27,13 +27,29 @@
 class CategoryManager(Manager):
 
     def prefetch_for_api(self):
+        # type: () -> django.db.models.QuerySet
+        """ Return Category QuerySet with added prefetch_related for api"""
         return self.get_queryset().prefetch_related(
             'products__images', 'products__variants',
             'products__variants__stock')
 
 
 @python_2_unicode_compatible
 class Category(MPTTModel):
+    """ Category Model, inheritance from MPTTModel
+
+    Fields:
+    name -- name of category (Char Field, max length 128)
```

Comment:
There should be some form of consistency  - if you're providing classes in brackets, do it in all classes